rest/route/task.go

```diff
@@ -418,6 +418,7 @@ func (rh *taskSyncPathGetHandler) Run(ctx context.Context) gimlet.Responder {
 
```

Comment:
I can't comment on unchanged lines, but the GET comment above this should be POST.

Reply:
Done.